webapi/src/main/scala/org/knora/webapi/messages/admin/responder/projectsmessages/ProjectsMessagesADM.scala

```diff
@@ -338,12 +341,18 @@ case class ProjectKeywordsGetResponseADM(keywords: Seq[String]) extends KnoraRes
  *
  * @param settings the restricted view settings.
  */
-case class ProjectRestrictedViewSettingsGetResponseADM(settings: ProjectRestrictedViewSettingsADM)
+case class ProjectRestrictedViewSettingsResponseADM(settings: ProjectRestrictedViewSettingsADM)
     extends KnoraResponseADM
     with ProjectsADMJsonProtocol {
   def toJsValue: JsValue = projectRestrictedViewGetResponseADMFormat.write(this)
```

Comment:
Given that we have a `zio-json` codec now it would be nice to look into removing the spray json format. Could you please look into that.

Reply:
No, this is out of scope. I renamed this class to reuse also with `POST` routes I implemented in this PR, but it ended up that it's not reuse, so I will bring old name back.

Reply:
`ProjectRestrictedViewSettingsResponseADM` and `ProjectRestrictedViewSettingsGetResponseADM` should be exactly the same response. I do not think having two classes for the identical response has some value for us.

I get your point, but I think we should explore the option of removing spray-json further. It seems like it could be a relatively simple task. Can you clarify why you think it's out of scope? It might not be as complicated as it initially sounds.

Reply:
There is only one class left, now called as it was before `ProjectRestrictedViewSettingsGetResponseADM`. This PR is about adding new routes, not refactoring entire package.

Reply:
I value your perspective, and I'd like to emphasize that my goal wasn't to kickstart a significant package refactoring. Instead, I aimed to make a minor yet valuable enhancement that could enhance code simplicity and long-term maintainability.

I'm sincerely disappointed that the suggestion to investigate removing spray-json wasn't taken into account. It appeared to be a manageable task with potential benefits for our codebase. While I'm disappointed, I believe that it's crucial to consider even smaller improvements in the spirit of maintaining a clean and efficient codebase. Despite this, I still approve this PR.